kalite/i18n/__init__.py

```diff
@@ -351,21 +351,6 @@ def set_default_language(lang_code):
     Settings.set("default_language", lcode_to_ietf(lang_code))
 
 
-def get_langs_with_subtitle(youtube_id):
```

Comment:
What's our strategy for maintaining this feature though? I'm assuming that now we don't show the available subtitles for a video, since this is deleted.